drivers/sensor/bh1749/bh1749_trigger.c

```diff
@@ -172,8 +173,7 @@ int bh1749_gpio_interrupt_init(struct device *dev)
 
 	if (gpio_pin_configure(drv_data->gpio,
 			DT_INST_0_ROHM_BH1749_INT_GPIOS_PIN,
-			(GPIO_DIR_IN | GPIO_INT | GPIO_INT_LEVEL |
-			GPIO_INT_ACTIVE_LOW | GPIO_PUD_PULL_UP))) {
+			(GPIO_INPUT | GPIO_PULL_UP))) {
```

Comment:
Using `GPIO_ACTIVE_LOW` here and `GPIO_INT_LEVEL_ACTIVE` in the interrupt configuration would be perhaps clearer.

Reply:
Same as for serial_lte_modem. I do not want to change actual implementation by the upmerge.